swap_meet/item.py

```diff
@@ -1,2 +1,28 @@
+from uuid import uuid4
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):
+        self.id = uuid4().int if id is None else id
+        self.condition = condition
+
+    def get_category(self):
+        return self.__class__.__name__
+    
+    def __str__(self):
+        return f"An object of type {self.get_category()} with id {self.id}."
+    
+    def condition_description(self):
+        if self.condition >= 4:
+            condition_str = "Near Mint! That's a good pick!"
+        elif self.condition >= 3:
+            condition_str = "Decent pick. Almost near mint."
+        elif self.condition >= 2:
+            condition_str =  "Moderately Used. It's aight."
+        elif self.condition >= 1:
+            condition_str = "You might want to reconsider."
+        else:
+            condition_str = "This is TRAASHHHHH!!"
+        return condition_str
```

Comment:
Nice job! Another way to approach this method would be to create a dictionary with the condition as keys and the string as values. Then instead of needing conditional statements (which are places that bugs could potentially get introduced), you could use self.condition to get the string value associated with the integer key. 

```python
condition_dict = {4: "Near Mint! That's a good pick!"}
condition_dict[self.condition]
```

This does mean that you wouldn't be able to use > or < but that logic could be handled by returning some default message if a number key isn't present.